### PR TITLE
Turn off multiprocessing for windows

### PR DESCRIPTION
**Pull request by vtomole**
Fixes: https://github.com/quantumlib/Cirq/issues/1541

Will remove "Travis windows" once approved.

### CODE REVIEW

File: cirq/google/sim/mem_manager_test.py [hunk 1]
```diff
@@ -79,6 +80,8 @@ def test_fills_gaps():
     mem_manager.SharedMemManager.free_array(new_handle)
 
 
+@pytest.mark.skipif(sys.platform.startswith('win'),
+                    reason="skipping linux-only tests")
```
Comment (dabacon) [inline cirq/google/sim/mem_manager_test.py]: "skipping linux-only tests" makes it sound like it is skipping linux tests. 
I'd make the reason something more about why this test
"skipping test on windows due to lack of multiprocessor support"

Comment (vtomole) [reply cirq/google/sim/mem_manager_test.py]: Done.

File: cirq/google/sim/mem_manager_test.py [hunk 2]
```diff
@@ -79,6 +80,9 @@ def test_fills_gaps():
     mem_manager.SharedMemManager.free_array(new_handle)
 
 
+@pytest.mark.skipif(sys.platform.startswith('win'),
+                    reason='Skipping test on Windows due'
```
Comment (dabacon) [inline cirq/google/sim/mem_manager_test.py]: nit space after 'due' so string doesn't say 'dueto'

Comment (vtomole) [reply cirq/google/sim/mem_manager_test.py]: Long day! Haha

**Pull request merged**